### DISCUSSION TITLE
Generic Plugin

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#146] [General]
Not sure if this is possible or not. Looking for something that I could use to load a small PHP application that lives in the root folder of a site into the admin area, maybe by iframe or something like that. So a user can login and access the "application" via the admin are of their website.

**Comment by guustnieuwenhuis** [comment]
With admin area, do you mean the Masa Administrator or a part of the website that requires login?

- Masa Administrator
You can do this with a custom plugin that has an interface and loads the PHP app in an iFrame

- Website
You can use the embed module to add an iFrame to the page that loads the PHP app

**Comment by brucebailey** [comment]
The Masa admin. I vaguely remember there used to be a plugin "template" running around. That should still work correct?

**Comment by guustnieuwenhuis** [comment]
I think you mean this one: https://github.com/stevewithington/MuraPlugin

I haven't tested it with Masa CMS (yet).

**Comment by brucebailey** [comment]
Yes, that is the one I was thinking of! I will download it and break it apart and see what is in there.

**Comment by brucebailey** [comment]
Just an update.. I pulled the plugin template apart from above and got it working with the iFrame. The only thing I could not get working is adding it to the Main Admin navigation on the left. It added it under plugins, but I could not get it working otherwise. Any suggestions from the group on how to make that happen?
